src/packages/solid/withScrolling.ts

```diff
@@ -0,0 +1,132 @@
+import { createEffect, on } from 'solid-js';
+import { activeElement, ElementNode } from '@lightningjs/solid';
+
+export function withScrolling(containerRef: ElementNode, props) {
+  let lastItemPosition,
+    lastItemSize,
+    windowVal,
+    prevIndex = 0,
+    componentRef,
+    updateIndex = false;
+
+  const [direction, prop, gap] = props();
+  const scrollType = formatScrollType(prop.scrollType);
+
+  createEffect(
+    on(
+      activeElement,
```

Comment:
Don't use activeElement change that is a global. Use onSelectedChanged which will only get called when the row is handling direction input